.github/workflows/rust.yml

```diff
@@ -10,11 +10,7 @@ jobs:
       - uses: actions/checkout@v4
       - name: Setup Rust Toolchain
         uses: ./.github/actions/setup-builder
-        with:
-          # Note that `nightly` is required for `license_template_path`, as
-          # it's an unstable feature.
-          rust-version: nightly
-      - run: cargo +nightly fmt -- --check --config-path <(echo 'license_template_path = "HEADER"')
+      - run: cargo fmt -- --check 
```

Comment:
It appears the option in question was removed some time ago from `rustfmt`: https://github.com/rust-lang/rustfmt/issues/5103

Thus let's simply simplify the CI rule